### PR TITLE
Register itsrohansahay.is-a.dev

### PR DESCRIPTION
**Pull request by srohansahay**
Register itsrohansahay.is-a.dev with CNAME record pointing to itsrohansahay.

### CODE REVIEW

File: domains/itsrohansahay.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "srohansahay",
+           "email": "f20201697@goa.bits-pilani.ac.in",
+           "discord": "783748496120872960"
+        },
+    
+        "record": {
+            "CNAME": "itsrohansahay"
```
Comment (CuteDog5695) [inline domains/itsrohansahay.json:9]: invalid CNAME record

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/itsrohansahay.json (hunk 1 above)
Comment (srohansahay) [reply domains/itsrohansahay.json:9]: hey can you tell me what a cname should be? My website is hosted on this[]( https://react-portfolio-vert-nu.vercel.app/)

Comment (CuteDog5695) [reply domains/itsrohansahay.json:9]: we dont support Vercel

Comment (CuteDog5695) [reply domains/itsrohansahay.json:9]: but you can use URL tho

Comment (srohansahay) [reply domains/itsrohansahay.json:9]: but i want .is-a.dev domain

Comment (CuteDog5695) [reply domains/itsrohansahay.json:9]: then use another hosting service, we dont support Vercel and Netlify

Comment (srohansahay) [reply domains/itsrohansahay.json:9]: mm can u pls suggest one hosting service?

Comment (CuteDog5695) [reply domains/itsrohansahay.json:9]: you can use Github Pages or Cloudflare Pages

Comment (srohansahay) [reply domains/itsrohansahay.json:9]: ok will try'em thanks!

**Pull request closed without merging**